oci-ignite-advanced/iam/identity-access-management-identity-domain.md

```diff
@@ -0,0 +1,216 @@
+
+* Your **<font color="red">Oracle Cloud Account</font>** - During this workshop we will create a basic environment for you to use on your tenancy.
+
+
+## Task 1: Create a Compartment
+
+A compartment is a collection of cloud assets, like compute instances, load balancers, databases, etc. By default, a root compartment was created for you when you created your tenancy (i.e. when you registered for the trial account). It is possible to create everything in the root compartment, but Oracle recommends that you create sub-compartments to help manage your resources more efficiently.
+
+1. Click the **Navigation Menu** in the upper left, navigate to **Identity & Security**, and select **Compartments**.
+
+ ![Navigation Menu](images/id-compartments.png " ")
+
+1. Click **Create Compartment**.
+   ![Create a compartment](images/create-compartment3.png)
+
+1. Name the compartment **Demo** and provide a short description. Be sure your root compartment is shown as the parent compartment. Press the blue **Create Compartment** button when ready.
+
+   ![Create compartment](images/compartment-details2.png)
+
+1. You have just created a compartment for all of your work in this *Workshop*.
+
+##  Task 2: Manage Users, Groups, and Policies to Control Access
+
+A user's permission to access services comes from the _groups_ to which they belong. The permissions for a group are defined by policies. Policies define what actions members of a group can perform, and in which compartments. Users can access services and perform operations based on the policies set for the groups of which they are members.
+
+We'll create a user, a group, and a security policy to understand the concept.
+
+In 2022, OCI IAM introduced Identity Domains. An identity domain is a container for managing users and roles, federating and provisioning users, secure application integration through Oracle Single Sign-On (SSO) configuration, and OAuth administration.
+
+1. Click the **Navigation Menu** in the upper left. Navigate to **Identity & Security** and select **Domains**
+
+For IAM with Identity Domains, what was identified before as IAM users and groups, now is under the default domain.
+
+   ![Navigate to Domains](images/id-domains.png)
+
+1. Select the default domain
+
+   ![Select the Default Domain](images/id-domains-default.png)
+
+1. Select **Groups**
+
+   ![Navigato to Groups](images/id-domains-groups.png)
+
+1. Click **Create Group**.
+
+   In the **Create Group** dialog box, enter the following:
+
+     - **Name:** Enter a unique name for your group, such as **oci-group**   
+     - **Description:** Enter a description, such as **New group for oci users**
+     - Click **Create**
+
+   ![Create Group](images/id-domains-create-group.png) 
+
+1. Click your new group to display it. Your new group is displayed.
+
+   ![New group is shown](images/id-domains-group-detail.png)
+
+1. Create a New User
+
+   a) In the breadcrumb, click on **Default Domain**
```

Comment:
We recommend only using numbers followed by dot( . ) or underordered lists using dashes ( - ) or asterisks ( * ). Please remove a) b) c) format

Reply:
done.